service/service.go

```diff
@@ -6,15 +6,14 @@ import (
 	"context"
 	"sync"
 
-	"github.com/giantswarm/apiextensions/pkg/clientset/versioned"
+	"github.com/giantswarm/apiextensions/pkg/apis/core/v1alpha1"
```

Comment:
I'd prefer to import this as corev1alpha1 but I'm not big on this.

Reply:
I think we occasionally have used `corev1` to alias `k8s.io/api/core/v1` - that gets potentially confusing quite quickly - WDYT?

Reply:
It doesn't confuse me but I get your point.